Libs/DICOM/Widgets/ctkDICOMVisualBrowserWidget.cpp

```diff
@@ -3139,7 +3231,7 @@ void ctkDICOMVisualBrowserWidget::onJobFailed(QList<QVariant> datas)
     {
       d->updateFiltersWarnings();
       d->SearchMenuButton->setIcon(QIcon(":/Icons/query_failed.svg"));
-      d->WarningPushButton->setText(tr("The patients query failed. Please check the servers settings."));
+      d->WarningPushButton->setText(ctkDICOMVisualBrowserWidget::tr("The patients query failed. Please check the servers settings."));
```

Comment:
Same as everywhere: `ctkDICOMVisualBrowserWidget::` is not needed here (only in `ctkDICOMVisualBrowserWidgetPrivate`)